builtin/settingtypes.txt

```diff
@@ -571,21 +571,30 @@ parallax_occlusion_bias (Parallax occlusion bias) float 0.04
 
 [***Waving Nodes]
 
-#    Set to true enables waving water.
+#    Set to true to enable waving liquids (like water).
 #    Requires shaders to be enabled.
-enable_waving_water (Waving water) bool false
+enable_waving_water (Waving liquids) bool false
 
-water_wave_height (Waving water wave height) float 1.0
+#    The maximum height of the surface of waving liquids.
+#    1.0 = Wave reaches the top of blocks.
+#    0.0 = Wave doesn't move at all.
+#    Requires waving liquids to be enabled.
+water_wave_height (Waving liquids wave height) float 1.0 0.0 1.0
 
-water_wave_length (Waving water wavelength) float 20.0
+#    Length of liquid waves.
+#    Requires waving liquids to be enabled.
+water_wave_length (Waving liquid wavelength) float 20.0 0.1
 
-water_wave_speed (Waving water wave speed) float 5.0
+#    How fast liquid waves will move. Higher = faster.
+#    If negative, liquid waves will move backwards.
+#    Requires waving liquids to be enabled.
+water_wave_speed (Waving liquid wave speed) float 5.0
```

Comment:
`liquid` or `liquids` as settings name? Needs consistency.

Reply:
Settings menu reads 'waving liquids' so i guess that.

Reply:
"Waving liquids" sounds better.